### PR TITLE
Add the first attempt for automatic evaluation

### PR DESCRIPTION
**Pull request by nclskfm**
# What does this PR do?

Add the first attempt for automatic evaluation:
* Implement async prediction
* Implement async evaluation
* Implement triggering evaluation after finishing the prediction
* Add tech stack: Celery, Redis, RabbitMQ

## Before submitting / marking as 'ready to review'
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

## Who can review?

- @HaritzPuerto
- @timbmg

### CODE REVIEW

File: evaluator/evaluator/app/core/task_helper.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        token (str): Authentication token. Required for non-public skills.
+
+    Returns: True if the skill exists. Otherwise False.
+    """
+    headers = {}
+    if token:
+        headers = {"Authorization": f"Bearer {token}"}
+
+    response = requests.get(
+        f"https://square.ukp-lab.de/api/skill-manager/skill/{skill_id}", headers=headers
```
Comment (timbmg) [inline evaluator/evaluator/app/core/task_helper.py:44]: Can you use an env variable for the host (https://square.ukp-lab.de/api) I think we use `SQUARE_API_URL`. This way, running it on a different domain will be easily configurable.

File: evaluator/evaluator/tasks/celery.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import logging
+import os
+
+from celery import Celery
+
+logger = logging.getLogger(__name__)
+
+# initialize env vars
```
Comment (timbmg) [inline evaluator/evaluator/tasks/celery.py:8]: why not use the /app/settings classes here too?

File: evaluator/evaluator/app/core/task_helper.py (hunk 1 above)
Comment (nclskfm) [reply evaluator/evaluator/app/core/task_helper.py]: Done ✅

File: evaluator/evaluator/tasks/celery.py (hunk 2 above)
Comment (nclskfm) [reply evaluator/evaluator/tasks/celery.py:8]: Hi, what do you mean? We took our orientation from the Celery scripts from you guys :D

like here: https://github.com/UKP-SQuARE/square-core/blob/master/square-model-inference-api/inference_server/tasks/celery.py#L1-L14 

```py


import logging
import os

from celery import Celery

logger = logging.getLogger(__name__)

# use env vars
rabbitmq_user = os.getenv("RABBITMQ_DEFAULT_USER", "ukp")
rabbitmq_password = os.getenv("RABBITMQ_DEFAULT_PASS", "secret")

redis_user = os.getenv("REDIS_USER", "ukp")
redis_password = os.getenv("REDIS_PASSWORD", "secret")
redis_host = os.getenv("REDIS_HOST", "redis")
```

Comment (nclskfm) [reply evaluator/evaluator/tasks/celery.py:10]: @timbmg updated :)

**Pull request merged**